cu-webteam/d8-platform
# Adds styles to fix panelized page issues

# Description

This removes padding on panelized page content so that it lines up with the body content.

## Related PRs

N/A

## Todos

- [x ] Tests - visually tested on local
- [x ] Documentation - N/A as this is a style change

## Deploy Notes

N/A

## Steps to Test or Reproduce

Open

1. Pull the panelized-pages branch to your local development environment
2. Copy the style.css to your clipboard
3. Visit a page with panelized content (http://publicsafety.test-creighton.acsitefactory.com/emergency-information/cu-safe)
4. In the web inspector, replace the content of the style.css with what's in your clipboard
5. Visually inspect the panelized content agains the body content to see how it aligns to the left.
6. Enter responsive design mode to test that the padding is the same as the menu above the content at mobile and tablet sizes.
